demo-app/src/commonMain/kotlin/dev/sargunv/maplibrecompose/demoapp/demos/ClusteredPointsDemo.kt

```diff
@@ -70,25 +72,21 @@ object ClusteredPointsDemo : Demo {
 
     val bikeSource =
       rememberGeoJsonSource(
-        "bikes",
         GeoJsonData.JsonString(data),
         GeoJsonOptions(
           minZoom = 8,
           cluster = true,
           clusterRadius = 32,
           clusterMaxZoom = 16,
-          // TODO on Android, this segfaults when the mapper is anything but a constant
-          // See https://github.com/maplibre/maplibre-native/issues/3493
-          // clusterProperties =
-          //   mapOf(
-          //     "total_range" to
-          //       GeoJsonOptions.ClusterPropertyAggregator(
-          //         mapper = feature["current_range_meters"].asNumber(),
-          //         reducer =
-          //           feature.accumulated().asNumber() +
-          //             feature["total_range"].asNumber(),
-          //       )
-          //   ),
+          clusterProperties =
+            mapOf(
+              "total_range" to
+                GeoJsonOptions.ClusterPropertyAggregator(
+                  mapper = feature["current_range_meters"].convertToNumber(),
+                  reducer =
+                    feature.accumulated().asNumber() + feature["total_range"].convertToNumber(),
+                )
+            ),
```

Comment:
I want to tweak this cluster property to maybe sum up the different _types_ of vehicles and show the sums in the cluster somehow. Will leave that for #461.

For now, it's enough that this no longer crashes.